code/extensions/che-on-start/package.nls.json

```diff
@@ -0,0 +1,4 @@
+{
+	"displayName": "Eclipse Che OnStart",
+	"description": "OnStart actions like project automatically at startup if no project is provided"
```

Comment:
Could you please rephrase it a bit, as it's hard to understand the extension's description

Reply:
updated